Sp4 polar plots

nkalscheuer opened the pull request:
Adding polar plot interpreter for evaluating optimal sail headings. If you have any questions let me know!

dan-rds commented:
- Lint as best as you can (I'll make a little guide for future reference)
- Might want to add something about where/how you git the csvs 
- It's not part of out def of done (because we don't have one) but I add [docstrings](https://www.python.org/dev/peps/pep-0257/) e.g. 
```python 
def bar(num):
    """
    bar() does ______
    Args:
        num (int): a number
    Returns:
        retVal (boolean): whatever
    """
```
or for shorter or less important functions, you can do a one line like so:
```python 
def bar(num):
    """ bar() does ______ and takes num """
```
Otherwise it looks good!!

nkalscheuer commented:
> * Lint as best as you can (I'll make a little guide for future reference)
> * Might want to add something about where/how you git the csvs
> * It's not part of out def of done (because we don't have one) but I add [docstrings](https://www.python.org/dev/peps/pep-0257/) e.g.
> 
> ```python
> def bar(num):
>     """
>     bar() does ______
>     Args:
>         num (int): a number
>     Returns:
>         retVal (boolean): whatever
>     """
> ```
> 
> or for shorter or less important functions, you can do a one line like so:
> 
> ```python
> def bar(num):
>     """ bar() does ______ and takes num """
> ```
> 
> Otherwise it looks good!!

- Linted the python file. Shows 10/10 for me now
- Added a readme about the csv info
- Added docstrings and more comments when doing lint. And added your sort of format for the doc strings
